core/MyTransport.cpp

```diff
@@ -825,11 +826,27 @@ void transportProcessMessage(void)
 #endif //defined(MY_OTA_LOG_RECEIVER_FEATURE)
 #if defined(MY_GATEWAY_FEATURE)
 		// Hand over message to controller
-		(void)gatewayTransportSend(_msg);
+		if (_msg.getType() == V_MULTI_MESSAGE) {
+			MyMessage msg2;
+			MyBlob blob(&_msg);
+			while (blob.getNext(msg2)) {
+				(void)gatewayTransportSend(msg2);
+			}
+		} else {
+			(void)gatewayTransportSend(_msg);
+		}
 #endif
 		// Call incoming message callback if available
 		if (receive) {
-			receive(_msg);
+			if (_msg.getType() == V_MULTI_MESSAGE) {
+				MyMessage msg2;
+				MyBlob blob(&_msg);
+				while (blob.getNext(msg2)) {
+					receive(_msg);
```

Comment:
This is a bug! 
Line 845 should read:
`receive(msg2);`

Reply:
With this change I was able to use this new MultiMessage N2N feature. Perfect!

Reply:
OMG, can't quite get into the mood these days and have little time available, I apologize for this.

Reply:
No problem, and now it is resolved. Thank you!